lib/dpl/provider/pypi.rb

```diff
@@ -66,7 +66,9 @@ def pypi_wheel_arg
       end
 
       def install_deploy_dependencies
-        unless context.shell "wget -O - https://bootstrap.pypa.io/get-pip.py | python - --no-setuptools --no-wheel && " \
+        # --user likely fails inside virtualenvs but is needed outside to avoid needing sudo.
+        unless context.shell "if [ -z ${VIRTUAL_ENV+x} ]; then export PIP_USER=yes; fi && " \
+                             "wget -nv -O - https://bootstrap.pypa.io/get-pip.py | python - --no-setuptools --no-wheel && " \
```

Comment:
What does `-nv` do?

Reply:
It disables the progressbar on wget download (cutting ~20 lines of useless console log).